packages/linux/package.mk

```diff
@@ -32,6 +32,12 @@ case "${LINUX}" in
     PKG_VERSION="6.4.8"
     PKG_SHA256="c59f34e19e84db30206b9373041abf893f9d8a08765d163586570a5238c458b6"
     PKG_URL="https://www.kernel.org/pub/linux/kernel/v${PKG_VERSION/.*/}.x/${PKG_NAME}-${PKG_VERSION}.tar.xz"
+    PKG_VERSION="024ff300db33968c133435a146d51ac22db27374"
+    PKG_SHA256=""
+    PKG_URL="https://github.com/torvalds/linux/archive/${PKG_VERSION}.tar.gz"
+    PKG_VERSION="6.5-rc7"
+    PKG_SHA256=""
+    PKG_URL="https://git.kernel.org/torvalds/t/${PKG_NAME}-${PKG_VERSION}.tar.gz"
```

Comment:
something gone wrong here ?

Reply:
No just me having all options in draft. Will be tiding it up this week - for release next week.